app/controllers/v1/EnterpriseUnitController.scala

```diff
@@ -2,14 +2,15 @@ package controllers.v1
 
 import javax.inject.{ Inject, Singleton }
 
-import play.api.libs.json.Json
-import play.api.mvc.{ Action, AnyContent, Controller, Result }
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import play.api.mvc.{ Action, AnyContent, Controller }
 import io.swagger.annotations._
 
 import uk.gov.ons.sbr.models.Period
 import uk.gov.ons.sbr.models.enterprise.{ Enterprise, Ern }
-import scala.concurrent.ExecutionContext.Implicits.global
 
+import controllers.v1.ControllerUtils._
 import repository.EnterpriseUnitRepository
 
 @Api("Search")
```

Comment:
Might be worthwhile adding a comment that explains that the validation is being undertaken by the router via regex constraints (see LocalUnitController).  No one will read it of course, but I'm conscious of the fact that it may not be easy for people to find how the validation is being done otherwise.